<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuScreenUI</class>
 <widget class="QDialog" name="MenuScreenUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addToCartButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add To Cart</string>
   </property>
  </widget>
  <widget class="QPushButton" name="toCheckoutButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>340</y>
     <width>151</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Proceed to Checkout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logOutButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Log Out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seeCartButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>View Cart</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
